src/bp/ui-studio/src/web/views/OneFlow/diagram/WorkflowToolbar.tsx

```diff
@@ -5,15 +5,36 @@ import { connect } from 'react-redux'
 import { flowEditorRedo, flowEditorUndo } from '~/actions'
 import { canFlowRedo, canFlowUndo } from '~/reducers'
 
+import style from './style.scss'
+
 const WorkflowToolbar = props => {
+  const { languages } = props
   const tabs = [
     {
       id: 'workflow',
       title: lang.tr('workflow')
     }
   ]
 
+  let languesTooltip = lang.tr('module.qna.form.translate')
```

Comment:
what does qna has to do with that ? I mean we should maybe move the translation / missing translations / everyting related to multilingual in it's own files ? or maybewe could have a shared thing

Reply:
my bad, copy and pasted and forgot to change it